### ISSUE TITLE
Improve c++ vtable parsing support

### ISSUE THREAD

**Issue by trufae** [radareorg/radare2#21975]
Use this doc as reference

https://hardwear.io/usa-2023/presentation/analyzing-decompiled-C++vtables-and-objects-in-GCC-binaries.pdf

**Comment by MewtR** [comment]
So I started looking into this a bit, was wondering if you had anything specific in mind. What would be the main improvement you'd like to see?

**Comment by trufae** [comment]
Yes, basically add support for what the document explains

**Comment by vingocheng** [comment]
I think different compiler have different vtable implement.

**Comment by trufae** [comment]
Yes, every compiler is differerent but at least for compat ressons c++ shares a bunch of abi stable agreements across different implementations but probably not for vtables
